contracts/PetPark.sol

```diff
@@ -1,7 +1,92 @@
 //SPDX-License-Identifier: Unlicense
 pragma solidity ^0.8.0;
-
+import "hardhat/console.sol";
```

Comment:
Suitable to always remove this from your final code because it costs a lot of gas if left during deployment